session.go

```diff
@@ -418,11 +422,27 @@ func (s *session) WriteBytes(pkg []byte) (int, error) {
 		return 0, ErrSessionClosed
 	}
 
-	lg, err := s.Connection.send(pkg)
+	totalSize := len(pkg)
+	if totalSize == 0 {
+		return 0, nil
+	}
+	writeSize := 0
+	s.wlock.Lock()
+	defer s.wlock.Unlock()
+	for totalSize >= maxPacketLen {
```

Comment:
totalSize change to leftPackageSize or packageSize should be more readable?

Reply:
addressed.